src/cincinnati/client.rs

```diff
@@ -34,6 +34,81 @@ pub struct Graph {
     pub edges: Vec<(u64, u64)>,
 }
 
+/// Cincinnati JSON protocol: service error.
```

Comment:
Ahh OK, I missed at first that this was actually part of the protocol itself until I saw the code deserializing straight from the response. Maybe link to the relevant part of the protocol here?

Reply:
You missed it because I really blurred it in the previous iteration. The whole protocol is linked in the top (module-level) docstring, is that enough?

Reply:
Ahh gotcha, yup that's fine!